packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeader.md

```diff
@@ -0,0 +1,38 @@
+---
+section: extensions
+subsection: Component groups
+id: Content header
+source: react
+propComponents: ['ContentHeader', 'ActionMenu']
+sourceLink: https://github.com/patternfly/react-component-groups/blob/main/packages/module/patternfly-docs/content/extensions/component-groups/examples/ContentHeader/ContentHeader.md
+---
+
+import ContentHeader from "@patternfly/react-component-groups/dist/dynamic/ContentHeader"
+
+The **content header** component displays a page title and description with a possibility of defining additional labels, links, or a page icon separated by a divider. 
+
+## Examples
+
+### Basic content header
+
+```js file="./ContentHeaderExample.tsx"
+
+```
+
+### Content header with breadcrumbs
+
+```js file="./ContentHeaderBreadCrumbExample.tsx"
+
+```
+
+### Content header with icon
+
+```js file="./ContentHeaderIconExample.tsx"
+
+```
+
+### Content header with label and link
+
+```js file="./ContentHeaderLabelLinkExample.tsx"
+
+```
```

Comment:
Can we add also an example for the actions?
Something like..

### Content header with action menu
In case you want to display actions in your header, you can use the `actionsMenu` property.

```js file="./ContentHeaderActionsExample.tsx"

```